examples/00-postprocessing/updated_script_manifold_example.py

```diff
@@ -135,35 +135,36 @@
 p_pathline.add_graphics(pathlines1)
 p_pathline.show()
 
-p_cont.plotter.view_isometric()
+p_cont._visualizer.plotter.view_isometric()
 
 p_surf = GraphicsWindow()
 p_surf.add_graphics(surface1)
 p_surf.show()
 
 
 def auto_refersh_call_back_iteration(session, event_info):
-    p_cont.refresh_windows(session.id)
-    p_res.refresh_windows(session.id)
-    p_mtr.refresh_windows(session.id)
-    p_mbr.refresh_windows(session.id)
+    p_cont.refresh(session.id)
+    p_res.refresh(session.id)
+    p_mtr.refresh(session.id)
+    p_mbr.refresh(session.id)
 
 
 def auto_refersh_call_back_time_step(session, event_info):
-    p_res.refresh_windows(session.id)
+    p_res.refresh(session.id)
 
 
 def initialize_call_back(session, event_info):
-    p_res.refresh_windows(session.id)
-    p_mtr.refresh_windows(session.id)
+    p_res.refresh(session.id)
+    p_mtr.refresh(session.id)
 
 
-cb_init_id = session.events.register_callback("InitializedEvent", initialize_call_back)
-cb_data_read_id = session.events.register_callback(
-    "DataReadEvent", initialize_call_back
-)
-cb_itr_id = session.events.register_callback(
+session.events.register_callback("InitializedEvent", initialize_call_back)
+session.events.register_callback("DataReadEvent", initialize_call_back)
+session.events.register_callback(
     "IterationEndedEvent", auto_refersh_call_back_iteration
 )
```

Comment:
@prmukherj I think we can avoid using strings here now--if you'd like to update that, it will look nicer.

Reply:
Sure @seanpearsonuk, will do that. This entire thing will be encapsulated now as well.

Reply:
Implemented the suggestions. Thank you.